Library/Homebrew/test/utils_test.rb

```diff
@@ -14,6 +14,14 @@ def teardown
     ENV.replace @env
   end
 
+  # Helper for matching escape sequences.
+  def e(code)
```

Comment:
Even although this is a test, let's use a longer name. Generally I think short name plus comment is worth than long name without comment.